### ISSUE TITLE
Define a blobstore for task scheduler in nexus3

### ISSUE THREAD

**Issue by fatemealizade** [ansible-ThoTeam/nexus3-oss#368]
Hi, As far as I saw in create_task.groovy file, there wasn't anything about blob store for task scheduler
how can I define a blob store for my task scheduler with this script?

**Comment by zeitounator** [comment]
Which kind of task are you trying to set up ? You can see a list of examples for the different tasks [en the wiki](https://github.com/ansible-ThoTeam/nexus3-oss/wiki/Scheduled-tasks-configuration#task-types-and-related-settings)

As an example [here are the examples for blobstore compaction](https://github.com/ansible-ThoTeam/nexus3-oss/wiki/%5BTask-Type%5D-Admin-%7C-Compact-blob-store)

**Comment by fatemealizade** [comment]
I want to set blob store for the task I'm creating
![11111111111](https://user-images.githubusercontent.com/53145698/212736907-9ae1a430-42e9-4f82-b9f6-5b05c4fb7208.png)
as you can see here in this image, there is a section about blobstore
But I didn't see anything about it in `create_task.groovy `

**Comment by zeitounator** [comment]
What isn't clear in the link I gave you above which is specifically for that exact example?

**Comment by fatemealizade** [comment]
It's an ansible file but I want it as a **groovy script**(Not that ansible file) which it's not defined in this [script](https://github.com/ansible-ThoTeam/nexus3-oss/blob/main/files/groovy/create_task.groovy) 
it only has a name, enabled, alertEmail and task frequency(i.e schedule) and I didn't anything about blobstoreName
I hope you get my point!

**Comment by zeitounator** [comment]
Sorry but no, I don't. I showed you above the way the script is called inside nexus from ansible and you just have to do the exact same thing from curl or wathever http client with the payload I also showed you as an example (which needs to be formated as json). The groovy script is here: https://github.com/ansible-ThoTeam/nexus3-oss/blob/main/files/groovy/create_task.groovy. It receives in `parsed_args` the payload I've linked with different examples already many times above. 

As explained in the wiki I also linked many times, the task specific properties are inside the `tasksProperties` and `booleanTasksProperties` keys inside each payload sent to the script and are then set inside the groovy script here:
https://github.com/ansible-ThoTeam/nexus3-oss/blob/3925e65d0dcb0d57086af53131d85b889da5da21/files/groovy/create_task.groovy#L34-L40

I don't know how I can be more clear and give more guidance. Good luck.
